app/scenes/AppNavigator.js

```diff
@@ -26,7 +26,8 @@ import Splash from './Splash';
 import VolumeInfo from './VolumeInfo';
 
 const Navigator = Actions.create(
-  <Lightbox>
+  // <Lightbox>
+  <Modal hideNavBar>
```

Comment:
I think we use the Lightbox when the user is logging out but has downloads in progress. Can't get rid of that completely although maybe we can move it to a scene or modal.